awx/sso/middleware.py

```diff
@@ -20,9 +18,12 @@
 
 class SocialAuthMiddleware(SocialAuthExceptionMiddleware):
 
-    def process_view(self, request, callback, callback_args, callback_kwargs):
-        if request.path.startswith('/sso/login/'):
```

Comment:
What's the deal with this being removed?  Did we try an SSO login after removing this?